Resources/Prototypes/_SpaceCats/Electromouse/gamerule.yml

```
@@ -0,0 +1,37 @@
+  - type: GhostRole
+    name: ghost-role-information-elmouse-name
+    description: ghost-role-information-elmouse-description
+    rules: ghost-role-information-elmouse-rules
+    raffle:
+      settings: default
+  - type: GhostRoleMobSpawner
+    prototype: MobElectroMouse
+  - type: Sprite
+    sprite: Markers/jobs.rsi
+    layers:
+      - state: green
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)